torch/csrc/jit/codegen/cuda/kernel_cache.cpp

```diff
@@ -566,7 +569,12 @@ std::vector<at::Tensor> FusionSegmentRuntime::runWithInput(
   // Produce final global output
   std::vector<IValue> fusion_outputs;
   for (auto output : segmented_fusion_->outputs()) {
-    fusion_outputs.push_back(tensor_map.at(output));
+    if (auto iter = tensor_map.find(output); iter != tensor_map.end()) {
+      fusion_outputs.push_back(iter->second);
+    } else {
```

Comment:
@shmsong 
This is the temporary hack I put there. I think it's safe as long as we insert a check here that we are only creating empty tensor for dangling TV.
But does it make more sense to have fusion segmentation to keep track of that?

Reply:
I see. I can add the dangling output TV's back into the segmented Fusion. The way it is now should work either way. Please feel free to add any additional checks that you'd like to do here and I can migrate that later.

Reply:
Thanks, I'll update this when merging fixes from dev_branch here.